content/blog/overview-of-the-platform-level-data-model-for-redfish®-device-enablement-standard.md

```diff
@@ -0,0 +1,158 @@
+
+Without the implementation of the PLDM for RDE standard, the MC (iLO) has
+a partial knowledge of external provider device properties and capabilities. This knowledge has been hardcoded in the Redfish service of the iLO firmware. As a consequence, Redfish clients can only get and set the properties referenced in the MC. For an exhaustive management of an external provider device, specific applications like the HPE Smart Storage Administrator (SSA) or the HPE MegaRaid Storage Administrator (MRSA) are needed.
+
+This paradigm (implementation of device properties in the iLO firmware) generates a lock step dependency between the iLO firmware and the device firmware; an update of the device firmware may lead to inaccurate iLO responses or errors if the iLO is not updated with a firmware embedding the new or modified device features.
+
+The picture below shows an iLO based server with two network adapters and one storage adapter. The iLO firmware is in sync with the storage device and one network adapter. This same level of firmware is represented with a green "F". The second network adapter firmware is not in sync with the iLO firmware and, is represented with a red "F". A red cross between the iLO and the un-synced firmware represents potential communication problems.
+
+In addition, this picture mentions the [Bus Master Interface Command](https://data.epo.org/publication-server/document?iDocId=698778&iFormat=0) (BMIC) and the Discovery and Configuration via iLO (DCi) protocols that are respectively patented and HPE proprietary protocols.
+
+Using proprietary protocols between the MC and an external provider device, require their implementation in both the MC and the external provider device. This is a constraint imposed to the external providers who have to implement all the specific protocols of all the computer makers they want to support.
+
+Moreover, device configuration is performed in two steps as explained in this [blog post](https://developer.hpe.com/blog/storage-management-with-redfish/): 1) the MC stores the modifications in a dedicated location. 2) The modifications are uploaded to the device during next server reboot. This reboot requirement is another drawback generated by the lack of PLDM implementation.
+
+![Operational model without PLDM for RDE](/img/withoutpldmrde.png "Operational model without PLDM for RDE")
+
+As a summary, the operational model without PLDM for RDE introduces the following problems:
+
+* firmware lock step dependency between iLO and devices
+* partial knowledge of device properties implemented iLO firmware
+* use of proprietary communication protocols between iLO and devices
+* Reboot required to load new device configuration settings
```

Comment:
either change "Reboot" to "reboot" or change all the other bullets to start with an initial capital letter